### PR TITLE
Issue #11489: Add LITERAL_TRY token support to WhitespaceAfter

### LINKED ISSUE #11489: WhitespaceAfter should support LITERAL_TRY token
https://checkstyle.org/config_whitespace.html#WhitespaceAfter

I have downloaded the latest cli from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

**How it works Now:**

```bash
/var/tmp $ javac TryStatement.java
/var/tmp $

/var/tmp $ cat config.xml
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="WhitespaceAfter">
            <property name="tokens" value="LITERAL_TRY"/>
        </module>
    </module>
</module>

/var/tmp $ cat TryStatement.java
import java.io.IOException;
import java.io.InputStream;

public class TryStatement {
    public static void main(String[] args) throws IOException {
        try(InputStream ignored = System.in) {

        }
    }
}

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-10.1-all.jar -c config.xml TryStatement.java
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - Token "LITERAL_TRY" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:476)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
        at com.puppycrawl.tools.checkstyle.Main.runCheckstyle(Main.java:403)
        at com.puppycrawl.tools.checkstyle.Main.runCli(Main.java:330)
        at com.puppycrawl.tools.checkstyle.Main.execute(Main.java:189)
        at com.puppycrawl.tools.checkstyle.Main.main(Main.java:126)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Token "LITERAL_TRY" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
        at com.puppycrawl.tools.checkstyle.TreeWalker.registerCheck(TreeWalker.java:223)
        at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:133)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:471)
        ... 5 more
Checkstyle ends with 1 errors.
```

**Is your feature request related to a problem? Please describe.**
It would be nice if checkstyle could detect missing space after `try` keyword.

**Describe the solution you'd like**
WhitespaceAfter could support new literal type - `LITERAL_TRY`

**Additional context**
Google Java Style Guide requires whitespace separation of `(` from any reserved words
See https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
>1. Separating any reserved word, such as `if`, `for` or `catch`, from an open parenthesis (`(`) that follows it on that line

#8122 seems related. This feature request is part of it.

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Resolves #11489 
Diff Regression config: https://gist.githubusercontent.com/Rahulkhinchi03/27b23af58be50c8ac5be9e3d44db6742/raw/288d528568ed0f9959360736b0abb63ff9df14a8/config.xml
Diff Regression patch config: https://gist.githubusercontent.com/Rahulkhinchi03/f6f4d6939dd8ac952b9fce093d33ea60/raw/47b475a36fadd791c6c3fe49972771829b7feaea/patch_config.xml

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespaceafter;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class InputWhitespaceAfterLiteralTry {
+
+        public static void main(String[] args) throws IOException {
+            // violation below ''try' is not followed by whitespace'
+            try(InputStream ignored = System.in) {
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java:17]: Put comment on same line.
Do indentation of file to be +2 to give you more space

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 2]
```diff
@@ -107,6 +109,30 @@
...
+ * <pre>
+ * &lt;module name=&quot;WhitespaceAfter&quot;&gt;
+ *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_TRY&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class TryStatement {
+ *     public static void main(String[] args) throws IOException {
+ *       try(InputStream ignored = System.in) { // violation ''try' is not followed by whitespace'
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:123]: Put this example to default config example.
As we add this token to default tokens.
No need for extra example section.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java (hunk 2 above)
Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java]: Changes Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java (hunk 1 above)
Comment (Rahulkhinchi03) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java]: Changes Done

File: src/xdocs/config_whitespace.xml [hunk 3]
```diff
@@ -2967,6 +2971,20 @@ public void myTest() {
     for (;;){}               // OK
     for(;;){}                // violation, space after 'for' is required
 }
+
+public class TryStatement {
```
Comment (romani) [inline src/xdocs/config_whitespace.xml:2975]: Put under "for"

Examples like
`try(InputStream ignored = System.in) {}`

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/config_whitespace.xml (hunk 3 above)
Comment (Rahulkhinchi03) [reply src/xdocs/config_whitespace.xml:2975]: Changes Done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 4]
```diff
@@ -84,7 +86,11 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *       try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *       try // OK
+ *       (InputStream ignored = System.in) {}
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:92]: Single line please, we need to show regular good code, not a edgecase

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespaceafter;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class InputWhitespaceAfterLiteralTry {
+    public static void main(String[] args) throws IOException {
+        try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java:15]: Put one more line with space after try

Comment (Rahulkhinchi03) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java:15]: Changes Done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java (hunk 4 above)
Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java]: Changes Done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 6]
```diff
@@ -84,7 +86,10 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *       try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *       try (InputStream ignored = System.in) {} // OK
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:91]: Please swap lines to match pattern from other, good goes first then bad

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java (hunk 5 above)
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java:15]: Keep example of separation by new line.
Iny comment I mentioned to add one more line to extend test cases

**Review by romani — APPROVED** [review]

**Comment by romani** [comment]
As we extend default, issue should have "new" label

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java [hunk 7]
```diff
@@ -26,4 +29,9 @@ public void check3 (int...a) { // warn
                 break;
         }
     }
+
+    public void check4() throws IOException {
+            try(InputStream ignored = System.in) { // warn
```
Comment (strkkk) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java:34]: please use fully qualified names to avoid import statements. They cause useless diff in violations, it is better to avoid such things

**Review by strkkk — CHANGES_REQUESTED** [review]
please see comment above

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java (hunk 7 above)
Comment (Rahulkhinchi03) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java]: Changes Done

**Review by strkkk — APPROVED** [review]

**Comment by nick-mancuso** [comment]
Github, generate report

**Comment by nick-mancuso** [comment]
Github, generate site

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+import java.io.InputStream;
+
+public class InputWhitespaceAfterLiteralTry {
+    public static void main(String[] args) throws IOException {
+        try (InputStream ignored = System.in) {} // OK
+
+        try // OK
+            (InputStream ignored = System.in) {}
+
+        try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
```
Comment (nick-mancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java:20]: Please add examples with traditional `try` block.

File: src/xdocs/config_whitespace.xml [hunk 9]
```diff
@@ -2966,6 +2970,9 @@ public void myTest() {
 
     for (;;){}               // OK
     for(;;){}                // violation, space after 'for' is required
+
+    try (InputStream ignored = System.in) {} // OK
+    try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
```
Comment (nick-mancuso) [inline src/xdocs/config_whitespace.xml:2975]: Please add examples with traditional `try` block.

**Review by nick-mancuso — CHANGES_REQUESTED** [review]
Items:

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java (hunk 8 above)
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java:20]: Try-catch

**Comment by nick-mancuso** [comment]
@Rahulkhinchi03 please add config and patch config to PR description and generate regression report.

**Comment by romani** [comment]
@Rahulkhinchi03, keep replying "done" as comment to each review item.

File: src/xdocs/config_whitespace.xml (hunk 9 above)
Comment (Rahulkhinchi03) [reply src/xdocs/config_whitespace.xml:2975]: Changes Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java (hunk 8 above)
Comment (Rahulkhinchi03) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralTry.java:20]: Changes Done

**Comment by romani** [comment]
GitHub, generate website

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 10]
```diff
@@ -84,7 +86,20 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *       try (InputStream ignored = System.in) {} // OK
+ *       try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *
+ *         try (InputStream ignored = System.in){} //OK
```
[the remark this answers is not in the document]
Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java]: Changes Done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 11]
```diff
@@ -84,7 +86,16 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *      try (InputStream ignored = System.in) {} // OK
+ *      try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *
+ *      try (InputStream ignored = System.in){} //OK
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:93]: Remove `()` block
https://docs.oracle.com/javase/tutorial/essential/exceptions/catch.html

It is not required try-catch.

It might allow you to make example code single line

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java (hunk 11 above)
Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java]: Please check if it correct now?

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 12]
```diff
@@ -84,7 +86,16 @@
...
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *      try (InputStream ignored = System.in) {} // OK
+ *      try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *
+ *      try {} //OK
+ *      catch(Exception exception){}
+ *
+ *      try{} // violation ''try' is not followed by whitespace'
```
[the remark this answers is not in the document]
Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:96]: Yes

[the remark this answers is not in the document]
Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java]: Changes Done

**Review by romani — APPROVED** [review]

**Comment by romani** [comment]
GitHub, generate website

**Review by pbludov — APPROVED** [review]

File: src/xdocs/config_whitespace.xml (hunk 9 above)
Comment (nick-mancuso) [reply src/xdocs/config_whitespace.xml:2975]: Resolved

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 13]
```diff
@@ -84,7 +86,13 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *      try (InputStream ignored = System.in) {} // OK
+ *      try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *
+ *      try {} catch(Exception exception){} // OK
```
Comment (nick-mancuso) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:93]: `exception` -> `e`

Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:93]: also, please add a space after catch (below line, too), this will become a violation after we add `LITERAL_CATCH` token, and we might forget to update examples.

**Review by nick-mancuso — CHANGES_REQUESTED** [review]
Last few items:

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java (hunk 13 above)
Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:93]: https://github.com/checkstyle/checkstyle/issues/11498

Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java]: Changes Done

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java [hunk 14]
```diff
@@ -26,4 +26,9 @@ public void check3 (int...a) { // warn
                 break;
         }
     }
+
+    public void check4() throws java.io.IOException {
+        try(java.io.InputStream ignored = System.in;) { // warn
+            }
```
[the remark this answers is not in the document]
Comment (Rahulkhinchi03) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java]: Changes Done

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java [hunk 15]
```diff
@@ -26,4 +26,8 @@ public void check3 (int...a) { // warn
                 break;
         }
     }
+
+    public void check4() throws java.io.IOException {
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java:30]: Please update InputWhitespaceAfterGood.java
To show what is good by Google

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java (hunk 14 above)
[the remark this answers is not in the document]
Comment (nick-mancuso) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java]: Resolved.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java (hunk 13 above)
Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java]: Resolved.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java [hunk 16]
```diff
@@ -84,7 +86,13 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
```
Comment (nick-mancuso) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Thanks for catching this, but please make this change in a separate commit `minor: formatting`

**Review by nick-mancuso — CHANGES_REQUESTED** [review]
One more:

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java (hunk 15 above)
Comment (Rahulkhinchi03) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfterBad.java:30]: Changes Done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java (hunk 16 above)
Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Okay, I am doing it.

Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Changes Done. Created new Commit.

Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Please make message of separate commit like example I gave above. Do not reference this issue, your change has nothing to do with it. This is why we are making a separate commit.

Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Also, your other commit should have only the change on this line in it, nothing else.

Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Changes Done

Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Not done, put only the change of indentation of closing `}` in separate commit.

Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: @nick-mancuso Just to clarify: I have to create a new commit in which `line 95:3` should just be formatted with `95:5`

Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: The closing brace of the method in this example was not indented correctly before your changes. While adding your changes to this example, you fixed the indentation of this closing brace. Fixing this indentation is not related to this issue. Any time that we make changes that are not related to the issue we work on, we do this in a separate commit. We do this to make review easier, and for future maintainers to have an easier time understanding changes that were made regarding a specific issue.

Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Please checkout now. I have done changes.

Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Not done. 

In first commit, make no changes to line 95. Only add to the code example (all changes “+” only in this code example).

In second commit, fix indentation of closing brace on line 95.

Comment (Rahulkhinchi03) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: done

**Review by romani — APPROVED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java (hunk 16 above)
Comment (nick-mancuso) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java:87]: Resolved.

**Review by nick-mancuso — APPROVED** [review]
